ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -192,6 +195,21 @@ class NotificationViewDemoController: DemoController {
                 notification.hide()
             }
             return notification
+        case .neutralToastWithGradientBackground:
+            let notification = MSFNotification(style: .neutralToast)
+            notification.state.message = "The background of this notification has been customized with a gradient."
+            notification.state.image = UIImage(named: "play-in-circle-24x24")
+            // It's a lovely blue-to-pink gradient
+            let colors: [DynamicColor] = [DynamicColor(light: ColorValue(0xFFF1FA), dark: ColorValue(0x601044)),
+                                          DynamicColor(light: ColorValue(0xE0F6FF), dark: ColorValue(0x005A7F))]
```

Comment:
Any chance we could get more readable colors?

Reply:
Your comment _does_ say it's a lovely blue-to-pink, but if we're getting the colors from somewhere, I'm curious what those are.

Reply:
I didn't do a full accessibility test, but to my eye the notification still looks pretty readable in both dark and light mode with these colors? What did you have in mind?

Reply:
Ah, by "readable", I meant not so much hex. I think the contrast is generally fine. Though maybe the dark mode blue could be a bit darker to match the dark mode blue of the primary style notifications

Reply:
Ahhh gotcha, sure!

Reply:
Using global colors now, with better dark mode contrast.